beacon_node/network/src/sync/manager.rs

```diff
@@ -949,6 +1038,68 @@ impl<T: BeaconChainTypes> SyncManager<T> {
         }
     }
 
+    fn on_single_data_column_response(
+        &mut self,
+        id: Id,
+        peer_id: PeerId,
+        data_column: RpcEvent<Arc<DataColumnSidecar<T::EthSpec>>>,
+    ) {
+        let Some((requester, resp)) = self
+            .network
+            .on_data_columns_by_root_response(id, data_column)
+        else {
+            // TOOD(das): error o log
+            return;
```

Comment:
Are you still thinking to add a log here? It looks like we don't log anything if block or blob requests aren't found.